_includes/assets/js/plugins/jquery.sdgMap.js

```diff
@@ -2,7 +2,11 @@
  * TODO:
  * Integrate with high-contrast switcher.
  */
-(function($, L, chroma, window, document, undefined) {
+(function($) {
+
+  if (typeof L === 'undefined') {
```

Comment:
Where does `L` point to here, if you removed it from the function's arguments?

Reply:
@SavvasStephanides That's a global variable created by Leaflet. It's not there, I assume that Leaflet did not get successfully loaded.

Reply:
Ah fair enough. Thanks!